src/session/index.ts

```diff
@@ -0,0 +1,87 @@
+//#region > Imports
+//#PACKAGE "js-cookie"
+//## npm install "js-cookie"@2.2.1
+// A simple, lightweight JavaScript API for handling browser cookies
+import Cookies from "js-cookie";
+//#endregion
+
+//#region > Interfaces
+/** @interface User defines the structure of a basic user object */
+interface User {
+  username: string;
+  password: string;
+}
+
+/** @interface ISession defines the session structure */
+interface ISession {
+  sessions: { [id: string]: ISession };
+  tokenName: string;
+}
+//#endregion
+
+//#region > Classes
+/** @class A general Session with token functionality */
+class Session implements ISession {
+  sessions: { [id: string]: ISession } = {};
+  tokenName: string = "token";
+
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {string} sId Session identifier
+   */
```

Comment:
Should have some sort of comment.

Initializes a base Session.

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3